pkg/validation/validate.go

```diff
@@ -1,31 +1,35 @@
 package validation
 
 import (
-	"path/filepath"
 	"strings"
 
 	"github.com/golang/glog"
 )
 
 const (
-	qcow2Ext = ".qcow2"
-	tar      = ".tar"
-	gzip     = ".gz"
-	xz       = ".xz"
+	qcow2 = ".qcow2"
+	tar   = ".tar"
+	tarGz = tar + ".gz"
+	tarXz = tar + ".xz"
 )
 
+// extList organizes the supported extensions for printing. When adding or removing an extension constant, make the
+// appropriate alteration in this slice.
+var extList = []string{qcow2, tar, tarGz, tarXz}
+
 func IsQcow2(file string) bool {
-	return filepath.Ext(file) == qcow2Ext
+	return strings.HasSuffix(qcow2, file)
 }
 
 func IsTarBall(file string) bool {
-	return strings.Contains(file, tar) && filepath.Ext(file) == gzip || filepath.Ext(file) == xz
+	return strings.HasSuffix(file, tar) || strings.HasSuffix(file, tarGz) || strings.HasSuffix(file, tarXz)
 }
 
 func IsValidImageFile(file string) bool {
+	f := strings.ToLower(strings.TrimSpace(file))
 	var valid bool
-	if valid = IsQcow2(file) || IsTarBall(file); !valid {
-		glog.Errorf("IsValidImageFile: file extension is unsupported. Expected *.qcow2, *.tar.gz or *.tar.xz; Got %s", file)
+	if valid = IsQcow2(f) || IsTarBall(f); !valid {
```

Comment:
nit: `main` logs the error if the extension is invalid, so we don't need 2 similar log entries. What about just returning:
`return IsQcow2(f) || IsTarBall(f)`